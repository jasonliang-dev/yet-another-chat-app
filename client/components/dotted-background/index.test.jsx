import React from 'react';
import { mount } from 'enzyme';
import DottedBackground from './index';

describe('<DottedBackground />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<DottedBackground />);
    expect(wrapper).toMatchSnapshot();
  });
});
