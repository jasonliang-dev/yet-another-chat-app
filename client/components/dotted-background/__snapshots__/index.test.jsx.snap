// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DottedBackground /> should match snapshot 1`] = `
<DottedBackground
  className=""
  style={Object {}}
>
  <div
    className=""
    style={
      Object {
        "backgroundImage": "url(test-file-stub)",
      }
    }
  />
</DottedBackground>
`;
